core/file_walker.go

```diff
@@ -60,9 +60,12 @@ func GetSubTree(path string, parent *File, readDir ReadDir, ignoredFolders map[s
 			files = append(files, file)
 		}
 	}
-	ret.Name = name
 	if parent != nil {
+		ret.Name = name
 		ret.Parent = parent
+	} else {
+		// Root dir
+		ret.Name = dirName + name
```

Comment:
Sorry for being late and leaving this project in a semi-broken state. IMO it doesn't make sense to see folders without having parsed them.
I'd merge this PR as-is, it's the easiest solution to the problem. What do you think?